packages/@sanity/form-builder/src/inputs/Slug/SlugInput.js

```diff
@@ -129,39 +134,19 @@ export default class SlugInput extends React.Component {
     }
     const {type, slugifyFn} = this.props
 
-    const slugify = get(type, 'options.slugifyFn') || slugifyFn
+    const slugify = get(type, 'options.slugify') || slugifyFn
```

Comment:
This would be a breaking change. Should be kept as is.

Reply:
ok. need to update the documentation then.